tox.ini

```diff
@@ -45,3 +45,13 @@ commands =
 whitelist_externals = make
 deps = -rtest_requirements.txt
 skip_install = true
+
+[testenv:safety]
+passenv =
```

Comment:
This is already defined in `[testenv]`

Reply:
Thanks @SharpEdgeMarshall, just updated the pr!